apps/web-mzima-client/src/app/shared/components/fragments/deployment-details/deployment-details.component.scss

```diff
@@ -17,10 +17,15 @@
       --size: 40px;
       --icon-size: 32px;
       margin-inline-start: 24px;
-      --color: var(--color-neutral-40);
-      --background-color: var(--color-black);
-      --border-color: var(--color-neutral-90);
-      --hover-background-color: var(--color-black);
+      &::ng-deep {
+        button {
+          border: 0.5px solid;
+          --color: var(--color-neutral-40);
+          --background-color: var(--color-black);
+          --border-color: var(--color-neutral-90);
+          --hover-background-color: var(--color-black);
+        }
+      }
 
```

Comment:
@oyenuga17 This method works actually, just that if you remove the close (and share) buttons and styles, the button styles is cleared up too. What you may want to do is declare these styles in the <mzima-client-button> component's styles, then add them to the button through fill and/or color attributes.
This would still help keep the styles.
This would be better helpful also when we start the restructuring for the styles.